presto-docs/src/main/sphinx/release/release-0.162.rst

```diff
@@ -0,0 +1,35 @@
+=============
+Release 0.162
+=============
+
+General Changes
+---------------
+* Fix correctness issue when the type of the value in the ``IN`` predicate does
+  not match the type of the elements in the subquery.
+* Fix correctness issue when the value on the left-hand side of an ``IN``
+  expression or a quantified comparison is ``NULL``.
+* Fix correctness issue when the subquery of a quantified comparison produces
+  no rows.
+* Fix correctness issue due to improper inlining of TRY arguments.
+* Fix correctness issue when the right side of a JOIN produces a very large
+  number of rows.
+* Fix correctness issue for expressions with multiple nested boolean
+  conjunctions and disjunctions.
+* Fix performance regression in constructing Hive table layouts.
+* Improve performance of window functions with similar ``PARTITION BY``
+  clauses.
+* Improve performance of certain multi-way JOINs by automatically choosing the
+  best evaluation order. This feature is turned off by default and can be
+  enabled via the ``reorder_joins`` config option or session property.
```

Comment:
The config property is ``reorder-joins`` and the session property is ``reorder_joins``